### ISSUE TITLE
A12Z ios16.5 Does anyone kfd succeed?  please tell us how you did it and set it up？

### ISSUE THREAD

**Issue by laaneo** [straight-tamago/misaka#81]
A12Z ios16.5 Does anyone kfd succeed?  please tell us how you did it and set it up？

**Comment by Git-Boy-Slim** [comment]
I also would be really interested in this kind of setup! New version should support those devices, but unfortunately I still get Kernel panic every time I try to exploit.

**Comment by felipebaldasso** [comment]
Same here. 11 pro max, 16.5 (20F65). On this new version 3.3.7, it's taking longer "exploiting" than the previous version, but it crashes after I click yes on KFD Mode.

**Comment by shanhanif1** [comment]
Yes crashing for me too on A12Z even though it says 3.6.6 added support for this cpu

**Comment by felipebaldasso** [comment]
I discovered something interesting: I open misaka, as soon as the app open I turn airplane mode on, close the app and open it again -- then, I see another message "Exploiting Kernel... Please wait..." instead "Exploiting". But it still crashes after a couple seconds 😅
